source/ios/AdaptiveCards/AdaptiveCards/AdaptiveCardsTests/Mocks/MockACRView.mm

```diff
@@ -0,0 +1,28 @@
+//
+//  MockACRView.m
+//  AdaptiveCardsTests
+//
+//  Copyright © 2021 Microsoft. All rights reserved.
+//
+
+#import "MockACRView.h"
+#import "MockContext.h"
+#import <Foundation/Foundation.h>
+
+@implementation MockACRView
+
+- (instancetype)initWithFrame:(CGRect)frame
+{
+    self = [super initWithFrame:frame];
+    return self;
+}
+
+- (void)setMockContext:(MockContext *)context
```

Comment:
>setMockContext [](http://example.com/codeflow?start=8&length=14)

This method is never called

Reply:
it's objective-c syntax. it's get called via @property